src/pages/upload/index.tsx

```diff
@@ -229,7 +235,7 @@ const Upload: React.FC<
               htmlFor="course-name"
               className="block text-sm font-medium text-gray-700"
             >
-              Course Name
+              Course Name*
```

Comment:
Is this a "required field" marker? Or just a stray character?

Reply:
I had that there to denote a required field